apps/studio/components/interfaces/Storage/StoragePolicies/StoragePolicies.tsx

```diff
@@ -176,67 +181,70 @@ const StoragePolicies = () => {
 
   return (
     <div className="flex min-h-full w-full flex-col">
-      <h3 className="text-xl">Storage policies</h3>
-      <p className="mt-2 text-sm text-foreground-light">
-        Safeguard your files with policies that define the operations allowed for your users at the
-        bucket level.
-      </p>
-
       {isLoading ? (
         <div className="flex h-full items-center justify-center">
           <Loader className="animate-spin" size={16} />
         </div>
       ) : (
-        <div className="mt-4 space-y-4">
-          {buckets.length === 0 && <StoragePoliciesPlaceholder />}
-
-          {/* Sections for policies grouped by buckets */}
-          {buckets.map((bucket) => {
-            const bucketPolicies = get(
-              find(formattedStorageObjectPolicies, { name: bucket.name }),
-              ['policies'],
-              []
-            ).sort((a: any, b: any) => a.name.localeCompare(b.name))
-
-            return (
+        <div>
+          <ScaffoldSection isFullWidth>
+            <ScaffoldSectionTitle>Buckets</ScaffoldSectionTitle>
+            <ScaffoldSectionDescription className="mb-6">
+              Write policies for each bucket to control access to the bucket and its contents
+            </ScaffoldSectionDescription>
+            {buckets.length === 0 && <StoragePoliciesPlaceholder />}
+
+            {/* Sections for policies grouped by buckets */}
+            {buckets.map((bucket) => {
```

Comment:
missing some gap in between
<img width="996" height="701" alt="image" src="https://github.com/user-attachments/assets/ad87f236-d764-4d2d-9a45-43091320699a" />